benchmark/convolution/conv2d.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import absolute_import, print_function, division
 import sys, timeit, time
-import numpy
+import numpy as np
```

Comment:
idem, the numpy in string here is code that is executed. So they need to be modified. Also not tested.